src/site/content/en/blog/pwas-on-oculus-2/index.md

```diff
@@ -0,0 +1,348 @@
+`ovr-platform-util` with the `create-pwa` command to generate the APK. The output file name can be
+set via the `-o` parameter, the Android SDK to use needs to be set via the `--android-sdk`
+parameter. Assuming a manifest file called `manifest.json` in the present working directory, the
+`--manifest-content-file` parameter helps the tool to determine the relevant configuration fields
+from the Web App Manifest. Alternatively, developers can point the tool at the live URL of a
+manifest via the `--web-manifest-url` parameter. To leave the manifest as pure as possible, the
+`--package-name` parameter can be used with a value in reverse domain name notation (e.g.,
+`com.company.app.pwa`), rather than adding the proprietary `ovr_package_name` field to the manifest.
+
+```bash
+ovr-platform-util create-pwa -o output.apk --android-sdk ~/bin/android-10 --manifest-content-file manifest.json --package-name com.company.app.pwa
+```
+
+### Installing PWAs with `adb`
+
+After creating the APK, developers can sideload the PWA with the `adb` command when the Oculus
+Quest&nbsp;2 device is connected via USB and when the connected computer is trusted. Sideloaded apps
+appear in an **Unknown Sources** section in the app drawer.
+
+```bash
+adb install output.apk
+```
+
+### Testing multi-tab apps
+
+To test multi-tab apps, I have created a little
+[test PWA](https://tomayac.github.io/oculus-pwa-test/) that demonstrates the various link features;
+namely opening a new in-PWA tab, staying on the current tab, opening a new browser window, and
+opening in a WebView staying on the current tab. Create a locally installable copy of this app by
+running the commands below on your machine.
+
+```bash
+ovr-platform-util create-pwa -o test.apk --android-skd ~/bin/android-10 --web-manifest-url https://tomayac.github.io/oculus-pwa-test/manifest.json --package-name com.example.pwa
+adb install test.apk
+```
+
+You can see a screencast of the test app in action embedded below.
+
+{% YouTube "3ZlxCjW9rtg" %}
+
+## An Oculus version of SVGcode
+
+To take the instructions for an actual spin, I have created an Oculus version of my most recent PWA,
+[SVGcode](/svgcode/). You can download the resulting APK file
+[`output.apk`](https://drive.google.com/file/d/1ieGjwIXGGWmh0j9WpBdWqP7Bns3McNr1/view?usp=sharing)
+from my Google Drive. If you want to investigate the package further, I have made available a
+[decompiled version](https://drive.google.com/drive/folders/1EFtXK9ApJiJitfysZS_Z7iIWWiKEglu-?usp=sharing),
+too. Find the build instructions in
+[`package.json`](https://github.com/tomayac/SVGcode/blob/6b23adf63a0a3a1b3828866dbb7db0f10206397f/package.json#L16).
+
+Using the app in VR works fine, including the ability to open and save files (though not using the
+[File System Access API](/file-system-access/), but using the
+[fallback approach](/browser-fs-access/#the-traditional-way-of-dealing-with-files)). The only thing
```

Comment:
This is an incomplete sentence. Please fix.

Reply:
Done.